sormas-app/app/src/main/java/de/symeda/sormas/app/backend/common/DatabaseHelper.java

```diff
@@ -2583,6 +2583,13 @@ public void onUpgrade(SQLiteDatabase db, ConnectionSource connectionSource, int
 				getDao(PathogenTest.class).executeRaw("UPDATE pathogenTest SET testedDiseaseVariant_id = NULL;");
 				getDao(Case.class).executeRaw("DROP TABLE diseaseVariant;");
 
+				case 302:
+					currentVersion = 302;
+					getDao(Case.class).executeRaw("ALTER TABLE cases ADD COLUMN internalToken text;");
+					getDao(Contact.class).executeRaw("ALTER TABLE contacts ADD COLUMN internalToken text;");
+					getDao(Person.class).executeRaw("ALTER TABLE person ADD COLUMN internalToken text;");
+					getDao(Event.class).executeRaw("ALTER TABLE events RENAME internalid TO internaltoken;");
```

Comment:
This doesn't work in sqlite.

You can keep the renamed field in the code and add `@Column(name = "internalid")` annotation.